src/App.scss

```diff
@@ -148,29 +149,29 @@ header {
 }
 
 @media (max-width: 700px) {
-    .landing-section > article > p {
+    .landing-section>article>p {
         padding: 0 40px;
     }
 }
 
 @media (max-width: 500px) {
-    .landing-section > article {
+    .landing-section>article {
         padding: 80px 20px;
     }
 
-    .landing-section > article > p {
+    .landing-section>article>p {
         padding: 0 20px;
     }
 }
 
-.landing-section > article > a {
+.landing-section>article>a {
     padding: 5px 10px;
     background: $green-color;
     color: $white-color;
     transition: 0.5s;
 }
 
-.landing-section > article > a:hover {
+.landing-section>article>a:hover {
```

Comment:
I do not like to not have spaces between the arrow, please, return the previous formatting.